### PR TITLE
Create mock api for events, roughly scaffold explore page

### PR DESCRIPTION
**Pull request by SonofNun15**
Added a mock api for events and put some rough scaffolding in place for the events page.

### CODE REVIEW

File: services/api/events.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      {
+        id: 1,
+        name: 'Leaf collection',
+        purpose: 'Rake and pickup leaves at the Lake Community park to tidy things up.',
+        minAttendance: null,
+        maxAttendance: null,
+        eventDate: 'Tomorrow at 10am',
+        organizer: {
+          name: 'Josh Graber',
+          gravatar: 'http://www.gravatar.com/avatar/5469091ba72e1841ae7868df28e5f648?s=48&d=identicon',
```
Comment (SeanMartz) [inline services/api/events.js:13]: Nice touch with the gravatar

**Review by SeanMartz — COMMENTED** [review]
looks good overall, do we want to focus on getting built for iOS and transition that to android when we're a little more comfortable? thoughts?

**Comment by SonofNun15** [comment]
I have an iPhone, so that was low hanging fruit for me. We probably do want to figure out if we can create views that are shared between them so that we can share some stuff and only duplicate the pieces that might be different.

**Comment by SonofNun15** [comment]
I want to figure out how to do icons next for the menu / header bar

**Review by SeanMartz — APPROVED** [review]

**Pull request merged**